tests/scripts/all.sh

```diff
@@ -1554,6 +1554,33 @@ component_build_crypto_full () {
   are_empty_libraries library/libmbedx509.* library/libmbedtls.*
 }
 
+component_build_crypto_light () {
```

Comment:
I didn't want to call it “crypto service” because it's a configuration of cryptography for a PSA service, not a configuration of the library as a service (which would require an IPC mechanism, and we don't have one in our code base). `crypto_for_psa_service` maybe?

I'll definitely change `build` to `test`.